### PR TITLE
ARM Template to create compute instances

### PR DESCRIPTION
**Pull request by ragargms**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

**Comment by ragargms** [comment]
@bmoore-msft for review

**Comment by ragargms** [comment]
@bmoore-msft ping!

**Comment by ragargms** [comment]
@bmoore-msft appreciate your review. We have a customer waiting on this.

File: 101-machine-learning-compute-create-computeinstance/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+            }
+        },
+        "adminPublicKey": {
+            "defaultValue": "",
+            "type": "securestring",
+            "metadata": {
+              "description": "The SSH public key of the administrator user account."
+            }
+        },
+        "sshPublicAccess": {
```
Comment (bmoore-msft) [inline 101-machine-learning-compute-create-computeinstance/azuredeploy.json:38]: what value is expected here?

File: 101-machine-learning-compute-create-computeinstance/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+            "type": "Microsoft.MachineLearningServices/workspaces/computes",
+            "name": "[concat(parameters('workspaceName'), '/', parameters('computeName'))]",
+            "apiVersion": "2020-04-01",
+            "location" : "[parameters('location')]",
+            "properties": {
+                "computeType": "ComputeInstance",
+                "properties":
+                {
+                    "vmSize" : "[parameters('vmSize')]",
+                    "subnet" : "[if(not(empty(parameters('subnetName'))), [resourceId(parameters('vnetResourceGroupName'), 'Microsoft.Network/virtualNetworks/subnets', parameters('vnetName'), parameters('subnetName'))], json('null'))]",
```
Comment (bmoore-msft) [inline 101-machine-learning-compute-create-computeinstance/azuredeploy.json:100]: fix this condition as in the other sample: https://github.com/Azure/azure-quickstart-templates/blob/master/101-machine-learning-compute-create-amlcompute/azuredeploy.json#L105

File: 101-machine-learning-compute-create-computeinstance/azuredeploy.json (hunk 1 above)
Comment (ragargms) [reply 101-machine-learning-compute-create-computeinstance/azuredeploy.json:38]: Enabled. Disabled. I will add that to the description.

File: 101-machine-learning-compute-create-computeinstance/azuredeploy.json (hunk 2 above)
Comment (ragargms) [reply 101-machine-learning-compute-create-computeinstance/azuredeploy.json:100]: fixed

**Comment by bmoore-msft** [comment]
@ragargms - check the deployment failure, looks like the scenario of enabling ssh access is not complete.

**Comment by ragargms** [comment]
@bmoore-msft  addressed comments

**Pull request merged**